src/tests/unit/sync/syncimages_ring.f90

```diff
@@ -0,0 +1,25 @@
+! SYNC IMAGES([this_image - 1, this_image + 1]) with the STAT=STAT_STOPPED_IMAGE
+! specifier and wrap around of image numbers.  The test is intended to check
+! that syncing in a ring with a stopped image still terminates all images.
```

Comment:
The motivation of the name is simple: It is testing the intrinsic statement `sync image` which should be modelled in the test's name. It is modelling a ring. That there is an abort to be expected can be reflected in the name: sync_image_ring_abort_on_stopped_image

Idea: A test sync_image_ring should be modelled @rouson where the abort is not expected and it is just tested whether a ring is syncing.

The code is just a copy from syncimage_status and I took over that style. If you want to polish the comments go ahead. Then I concentrate on fixing issues and adding functionality. It is always a good idea when the tests are not written by the dev who implemented a feature, because those tests tend to comply, because the dev knows what to do.